scripts/temperature2

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Depends on lm-sensors (https://packages.ubuntu.com/bionic/lm-sensors)
+
+VALUE_FONT=${font:-$(xrescat i3xrocks.value.font "Source Code Pro Medium 13")}
+
+TEMP=$(sensors -u | grep -Po 'temp1_input: \K\d*')
```

Comment:
This adds a dependency on lm-sensors.  Happily it looks like this package is well supported in Ubuntu: https://launchpad.net/ubuntu/+source/lm-sensors/1:3.5.0-3ubuntu1